modules/components/module.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace Elementor\Modules\Components;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Plugin;
+use Elementor\Modules\Components\Styles\Component_Styles;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+	const EXPERIMENT_NAME = 'e_components';
+	const PACKAGES = [ 'editor-components' ];
+
+	public function get_name() {
+		return 'components';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		if ( ! self::is_active() ) {
+			return;
+		}
```

Comment:
I am not sure this code is required when using the `get_experimental_data` method